app/ui-message/client/message.css

```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)